source/_components/binary_sensor.bayesian.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: "Bayesian Binary Sensor"
+description: "Instructions how to integrate threshold Bayesian sensors into Home Assistant."
+date: 2017-08-27 20:05
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Binary Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.53
+---
+
+
+The `bayesian` binary sensor platform observes the state from multiple sensors
+and uses Bayes' rule to estimate the probability that an event has occurred
+given the state of the observed sensors. If the estimated posterior probability
+is above the `probabiliy_threshold`, the value of the sensor is `on`.
+Otherwise, the sensor is `off`.
+
+This allows for the detection of complex events that may not be readily
+observable, i.e. cooking, showering, in bed, start of morning routine, etc.  It
```

Comment:
**Spelling & Grammar**:
- Missing comma after `i.e.` -> `i.e., cooking, showering`
- `start of morning routine` -> `the start of a morning routine`